src/swish/errors.ss

```diff
@@ -96,6 +99,7 @@
       [no-process "No process."]
       [timeout "Timeout."]
       [unexpected-eof "Unexpected end-of-file."]
+      [expected-dictionary "Expected dictionary."]
```

Comment:
Please move expected-dictionary immediately below content-limit-exceeded to keep the list sorted.

Reply:
Fixed.